src/commands/app/build.js

```diff
@@ -70,22 +70,24 @@ class Build extends BaseCommand {
     }
 
     if (flags.actions) {
-      if (config.app.hasBackend && (flags['force-build'] || !fs.existsSync(config.actions.dist))) {
+      // removed flags['force-build'] || as it is always forced at this point, we need to check to decide what to build
+      // if no backend, we skip the build
+      if (config.app.hasBackend) { //  && (flags['force-build'] || !fs.existsSync(config.actions.dist))) {
```

Comment:
q: I know this was here before but `flags.actions && !config.app.hasBackend` should just exit right?

Reply:
yeah, I need to clean this up, if there is no backend then there is nothing to build so we simply output 'no backend or a build already exists' but it should probably just be 'nothing to build, no backend'

Reply:
cleaned up!